app/bundles/LeadBundle/Form/Type/LeadType.php

```diff
@@ -209,5 +211,21 @@ public function configureOptions(OptionsResolver $resolver)
         );
 
         $resolver->setRequired(['fields', 'isShortForm']);
+
+        $resolver->addNormalizer('constraints', static function (Options $options, array $constraints): array {
```

Comment:
We are discussing over and over that we should move all constraints from forms to entities so the API wouldn't have to boot the Symfony Forms to validate entities. That is very expensive. I'd like to start by at least adding new constraints to entities over forms. What do you think?

Reply:
Here are some examples of adding constraints to entities:

https://github.com/mautic/mautic/search?q=loadValidatorMetadata

`$entityManager->clear()` doesn't do the job? Although it is dangerous to call it in the app. We do it a lot in tests to separate requests/commands.

Reply:
The issue was that the validator metadata is cached. Gone the way you can see in latest commit.